Source/Core/Gen5/Searchers/StationarySearcher5.cpp

```diff
@@ -18,24 +18,19 @@
  */
 
 #include "StationarySearcher5.hpp"
+#include <Core/Enum/Game.hpp>
+#include <Core/Enum/Method.hpp>
 #include <Core/Gen5/Keypresses.hpp>
 #include <Core/RNG/SHA1.hpp>
+#include <Core/Util/Utilities.hpp>
 #include <future>
 
-StationarySearcher5::StationarySearcher5(const StationaryGenerator5 &ivGenerator, const StationaryGenerator5 &pidGenerator,
-                                         const Profile5 &profile, const std::vector<std::unordered_map<u32, u32>> &ivMap, bool includePID) :
-    ivGenerator(ivGenerator),
-    pidGenerator(pidGenerator),
-    profile(profile),
-    ivMap(ivMap),
```

Comment:
FYI, IV stuff will need to be added back eventually to support fast IV search. Also I think it is worth discussing if searcher should be allowed to do PID and IVs separately or if it should only search them together. I like the idea of only together.

Reply:
Understandable to have them in searcher together. Since this is mainly a model issue, I personally don't see it as a priority to do just yet as we want to make sure they are generally working. Testing purposes it's easier to see when separate. I'm with the idea of merging them once the core is done.